src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1525,68 +1534,461 @@ static void LoadRegPair(BYTE* p, int reg1, int reg2, UINT32 offset)
 
 PCODE DynamicHelpers::CreateHelper(LoaderAllocator * pAllocator, TADDR arg, PCODE target)
 {
-    _ASSERTE(!"RISCV64: not implementation on riscv64!!!");
-    return NULL;
+    STANDARD_VM_CONTRACT;
+
+    BEGIN_DYNAMIC_HELPER_EMIT(32);
+
+    *(DWORD*)p = 0x00000297;// auipc t0, 0
+    p += 4;
+    *(DWORD*)p = 0x0102b503;// ld a0, 16(t0)
+    p += 4;
+    *(DWORD*)p = 0x0182b283;// ld t0, 24(t0)
+    p += 4;
+    *(DWORD*)p = 0x00028067;// jr t0
+    p += 4;
+
+    // label:
+    // arg
+    *(TADDR*)p = arg;
+    p += 8;
+    // target
+    *(PCODE*)p = target;
+    p += 8;
+
+    END_DYNAMIC_HELPER_EMIT();
```

Comment:
BTW, rather than manually calculate proper sizing and offsets each time, it could be useful to employ the compiler to do it for you, e.g. something like:
```c++
struct Asm
{
    DWORD instructions[4] = {
        UTypeInstr(AUIPC, t0, 0),
        ITypeInstr(LOAD, LD, a0, t0, offsetof(Asm, _arg)),
        ...
    };
    PCODE _arg = arg;
    PCODE _target = target;
};
BEGIN_DYNAMIC_HELPER_EMIT(sizeof(Asm));
*(Asm*)p = Asm{};
END_DYNAMIC_HELPER_EMIT();
```

Reply:
I left the logic of the functions by analogy with other architectures.